python/tank/authentication/resources/build_resources.sh

```diff
@@ -17,37 +17,64 @@ if [ -z "${PYTHON_BASE}" ]; then
     PYTHON_BASE="/Applications/Shotgun.app/Contents/Resources/Python"
```

Comment:
When building from MacOS, the `PYTHON_BASE` path is usually `/Applications/Shotgun.app/Contents/Resources/Python3/`